src/utils/constants.ts

```diff
@@ -13,7 +18,7 @@ export const FEIDE_IDENTITY_PROVIDER = 'FeideIdentityProvider';
 export const AMPLIFY_REDIRECTED_KEY = 'amplify-redirected-from-hosted-ui';
 export const REDIRECT_PATH_KEY = 'redirect-path';
 
-export const API_URL = process.env.REACT_APP_API_URL;
+export const API_URL = hostToUrl(process.env.REACT_APP_API_HOST);
```

Comment:
axois config vil ha `baseURL` som string, (ikke URL), så kan like godt sette det direkte her, og fjerne `hostToUrl`:
```suggestion
export const API_URL = `https://${process.env.REACT_APP_API_HOST}`;
```

Reply:
fixed. I want to keep the check of whether it starts with http or not because if the input at some point starts with https, I do not want to have https://https://www.example.com